### DISCUSSION TITLE
Recommended way to configure networking?

### DISCUSSION THREAD

**Discussion by paulmenzel** [systemd/mkosi#2597] [Q&A] [answered]
To configure systemd-networkd inside a virtual machine, is placing the network configuration file in `mkosi.extra/`, for example, mkosi.extra/etc/systemd/network/80-dhcp.network` the recommended/idiomatic way to do it?

    $ more mkosi.extra/etc/systemd/network/80-dhcp.network
    [Match]
    Name=en*
    
    [Network]
    DHCP=yes

**Comment by daandemeyer** [comment]
There is no preferred way, It's up to you to decide how you want to do networking in your image. If you want my personal opinion, I would myself use systemd-networkd.

**Reply by paulmenzel** [reply]
Understood. Thank you for the instant reply. Should that go into `/etc/` or to have an empy `/etc/` somewhere else?

**Reply by daandemeyer** [reply, marked as answer]
You can put in /usr/lib/systemd/network

**Reply by paulmenzel** [reply]
Thank you.
